openface:
  default_param:
    rate_frame: 1
    fps: 1
    window_size: 1
    overlap: 0
    executable_dir: $(find harmoni_openface)/OpenFace/build/bin/FaceLandmarkImg
    input_dir: $(find harmoni_openface)/input/
    output_dir: $(find harmoni_openface)/output/
    subscriber_id: default
    robot_subscriber_id: /camera/color/image_raw